content/integrations/agents/windows-agent/windows-agent-send-attributes.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Tags & Attributes"
+#date: 2018-12-11
+draft: false
+tags: ["#windows", "#integrations", "#install", "#tags", "#attributes", "#elements"]
+author: Lawrence Lane
+---
+
+You can send tags and attributes through the Windows Agent by updating files in from the `/CollectedWin/config` folder.
+
```

Comment:
Is it worth explaining the difference between a tag and an attribute?

Why would you want to send a tag vs an attribute?